data/sql/updates/pending_db_world/dustbelchernpcfix

```diff
@@ -0,0 +1 @@
+UPDATE `creature` SET `MovementType` = 1 WHERE `guid` IN (6974,6989,7210); -- correcting MovementType for creatures Dustbelcher Mauler, Dustbelcher Wyrmhunter, and Dustbelcher Shaman
```

Comment:
When you update the creature table, remember to add the `id1` as well
```
WHERE `guid` IN (6974,6989,7210) AND `id1` IN (X,Y,Z);
```
Also that comment in not strictly needed as the PR description already covers everything